### DISCUSSION TITLE
Use BSData Warcry

### DISCUSSION THREAD

**Discussion by Dreadaxe** [SmitdCatchy/warcry-companion#33] [Ideas]
Possibility to import BSData Warcry?
https://github.com/BSData/warhammer-age-of-sigmar-warcry

**Comment by SmitdCatchy** [comment]
That is a great idea! I will check out the BS data format. Until then you can use the export/import features in the app 😉

**Reply by Dreadaxe** [reply]
Excellent news!

**Reply by SmitdCatchy** [reply]
Bad news is that I cannot develop a BSData import, because the BSData files are not structured in a consistent manner. 😓
Good news there is a facebook group where you can share the warband exports: https://www.facebook.com/groups/791866102144880 and abything related to the app 😉
